index.html

```diff
@@ -0,0 +1,222 @@
+                                Stories</h1>
+                              <div class="work-role">
+                                <h3 class="work-role-list">CANOPY</h3>
+                                <div class="work-role-list role-dot"></div>
+                                <div class="work-role-list">Back end dev</div>
+                                <div class="work-role-list role-dot"></div>
+                                <div class="work-role-list">2023</div>
+                              </div>
+                              <p class="project-description">
+                                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                </p>
+                                <ul class="my-skill">
+                                    <li><button title="html">html</button></li>
+                                    <li><button title="css">css</button></li>
+                                    <li><button title="js">Javascript</button></li>
+                                </ul>
+                                <span class="p-more-information">
+                                  <a class="moresee-link btn-action-event">See project</a>
+                                  </span>
+                            </span>
+          </div>
+</section> 
+<section class="aboutmyself-section">
+    <div id="aboutmyself-container">
+      <span class="aboutmyself-content">
+<h1 class="aboutmyself-title intro-name">About<br>
+    Myself</h1>
+    <p class="aboutmyself-history intro-position">
+        I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! If you like what you see 
+        and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <p><br><a class="intro-connect btn-action-event">LET’S CONNECT</a><br></p>
+    <ul class="view-social-midea">
+        <li class="social-icon">
+            <a class="btn-action-event"><img src="./assets/images/social-icon/twitter-icon.png" alt="twitter"></a>
+        </li>
+        <li class="social-icon">
+            <a class="btn-action-event"><img src="./assets/images/social-icon/linkedin icon.png" alt="linkedin"></a>
+        </li>
+        <li class="social-icon ">
+            <a><img src="./assets/images/social-icon/m-icon.png" alt="m"></a>
+        </li>
+        <li class="social-icon ">
+            <a><img src="./assets/images/social-icon/github-icon.png" alt="github"></a>
+        </li>
+        <li class="social-icon ">
+            <a><img src="./assets/images/social-icon/clip-icon.png" alt="fingers"></a>
+        </li>
+    </ul>
+    <span class="p-more-information ">
+        <a class="moresee-link btn-action-event">Get my resume</a>
+        </span>
+        <ul class="expand-ul"><li><a class="expand-ula">Languages</a></li><li><input type="image" src="./assets/images/arrowdown.png" class="bn-img-down"></li></ul>
```

Comment:
- Well done adding the icon that the reviewer asked for 👏However the look is still different from what the Figma design is displaying 

### Your Design 
![image](https://user-images.githubusercontent.com/68030297/219854071-42cc1c32-75b1-49e6-957f-f8abfcf3850f.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/219854088-7e296941-fd4d-46b5-88a2-978c4fc03e55.png)

- [ ] Please ensure to minimize the button arrow down icon to look like the Figma design as much as possible 💪